CONTRIBUTING.md

```diff
@@ -2,78 +2,91 @@
 
 We welcome all types of contribution.
 
-Need a feature or found a bug? Please create an [issue.](https://github.com/iTwin/iTwinUI-react/issues)
+Need a feature or found a bug? Please create an [issue](https://github.com/iTwin/iTwinUI-react/issues).
 
-Want to contribute by creating a PR? Great! Then read further.
+Have a question or suggestion? Please create a [discussion](https://github.com/iTwin/iTwinUI-react/discussions).
 
-## Getting Started
+Want to contribute by creating a pull request? Great! [Fork iTwinUI-react](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/working-with-forks) to get started.
 
-We use yarn for package management, so be sure to have it installed
-`npm install -g yarn`
+---
+
+## How to setup
+
+To clone and build iTwinUI-react, you'll need [Git](https://git-scm.com) and [Yarn 1](https://yarnpkg.com/getting-started/install) installed on your computer.
 
-1. `git clone https://github.com/iTwin/iTwinUI-react.git`
-2. `cd iTwinUI-react`
-3. `yarn install`
+1. [Create a local clone](https://docs.github.com/en/github/getting-started-with-github/fork-a-repo#step-2-create-a-local-clone-of-your-fork) of your forked repository. You can do this from the command line or using the Github Desktop app.
+2. Go to the directory where you cloned iTwinUI-react. e.g. `cd iTwinUI-react`.
+3. Run `yarn install` from that directory.
 
 If using vscode, our prettier and editor configs will be used.
 Please make sure to install all recommended extensions in [extensions.json](./.vscode/extensions.json).
 
-## Storybook
+### Preview using Storybook
 
 We use [Storybook](https://storybook.js.org) to test and demo the components. You can run it locally with `yarn storybook`.
 Be sure to add stories for your newly added component or feature. Read more [here](./stories/README.md) about storybook and how we write stories.
 
+---
+
 ## Developing
 
-Before developing, please read our [style guide](./STYLEGUIDE.md)
+Before developing, please read our [style guide](./STYLEGUIDE.md).
 
 If you are creating a new component, use this script:
 
 `yarn createComponent`
 
 It ensures all needed imports are added and files are created.
 
-> Note: Every component needs to use `useTheme()` hook to ensure styling (theming) works fine. The script mentioned above adds it automatically.
-
-Be sure to develop on your branch. We want to keep branch naming with developer name scope:
+> Note: Every component needs to use `useTheme()` hook to ensure styling (theming) works fine. The `createComponent` script mentioned above adds it automatically.
 
-`git checkout -b yourName/your-feature-name`
```

Comment:
Sure we dont want to say anything about branch name?

Reply:
I think they can name their branch whatever they want. It's their fork 😄